mwdb/core/search/fields.py

```diff
@@ -152,6 +152,32 @@ def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
             return self.column.any(self.value_column == value)
 
 
+class FavoritesField(BaseField):
+    def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
+        if remainder:
+            raise FieldNotQueryableException(
+                f"Field doesn't have subfields: {'.'.join(remainder)}"
+            )
+
```

Comment:
Wildcard should not be accepted here (see "Wildcards are not allowed").

Future note: It should be refactored a bit in future PR to avoid copy-pasting these conditions everywhere (`accepts_wildcards`, `accepts_remainder`, similar to `accepts_range` implementation)